wrappers/pointcloud/pointcloud-stitching/doc/pointcloud-stitching-demo.md

```diff
@@ -0,0 +1,189 @@
+
+## Running rs-pointcloud-stitching
+
+### __Setting configuration files__
+1. **Defining the input stream:**</br>
+    For each connected device we need a configuration file to describe the input streams.</br>
+    The name of the file will be <serial_number>.cfg</br>
+    As mentioned in the calibration section, in this example I used 2 x D435i with the following serial numbers: 912112073098, 831612073525 using a resolution of 640x480 pixels for depth and 640x360 for color at 30 fps. Make sure you use the same resolution by which you calibrated the cameras.
+    If you reuse the files created for the gathering of images process, you can just remark the INFRARED streams using the number sign (#).</br>
+    The files should look as follows:
+
+    *912112073098.cfg:*</br>
+    ```
+    DEPTH,640,480,30,Z16,0
+    COLOR,640,360,30,RGB8,0
+    #INFRARED,640,480,30,Y8,1
+    ```
+
+    *831612073525.cfg:*</br>
+    ```
+    DEPTH,640,480,30,Z16,0
+    COLOR,640,360,30,RGB8,0
+    #INFRARED,640,480,30,Y8,1
+    ```
+
+2.	**A calibration file.**</br>
+    - The name of the file is unimportant as you specify it in `rs-pointcloud-stitching` command line.</br>
+    - It should contain 2 lines. One contains the calibration matrix between camera A and camera B. The other, the calibration matrix between camera B and the virtual camera that the app creates.</br>
+    - The transformation parameters are 9 rotation and then 3 translation parameters.
+
+    </br>
+    As an example, a calibration file describing 2 cameras with 15 degrees between them is given:</br>
+
+    *calibration_15.cfg*:
+    ```
+    912112073098, 831612073525, 0.8660254, 0,  -0.5,  0,1,0,      0.5, 0., 0.8660254, 0,0,0
+    831612073525, virtual_dev, 0.96592583, 0,  0.25881905,  0,1,0,      -0.25881905, 0., 0.96592583, 0,0,0
+    ```
+    **Notice:** "virtual_dev" is a key word.
+
+    Following the suggested calibration process, a calibration file was already created by export_calibratrion.m (C:\pc_stitching_ws\calibration_60m.cfg in this example).
+
+    *calibration_60m.cfg:*
+    ```
+    912112073098, 831612073525, 0.488316, -0.007477, -0.872635, 0.004309, 0.999972, -0.006157, 0.872656, -0.000754, 0.488334, 0.199384, 0.001205, -0.090225
+    831612073525, virtual_dev, 0.862643, 0.003806, 0.505799, -0.004022, 0.999992, -0.000666, -0.505797, -0.001460, 0.862651, -0.099692, -0.000602, 0.045112
+    ```
+
+    The first line describes the calibration from camera 912112073098 to 831612073525.</br>
+    The second line describes the calibration from 831612073525 to the virtual device.
+ 
+
+3.	**Defining the virtual device:**</br>
+rs-pointcloud-stitching requires a file defining the virtual device created by combining the other 2 devices.</br>
+It should contain wanted FOV and resolution for the virtual device.</br>

```

Comment:
Specify the FOV range/limitations that the user should be aware of

Reply:
I added according to the next remark